### PR TITLE
add 3.26.1 changelog

### PR DESCRIPTION
**Pull request by yesmeck**
-----
[View rendered CHANGELOG.en-US.md](https://github.com/ant-design/ant-design/blob/release-3.26.1/CHANGELOG.en-US.md)
[View rendered CHANGELOG.zh-CN.md](https://github.com/ant-design/ant-design/blob/release-3.26.1/CHANGELOG.zh-CN.md)

### CODE REVIEW

File: CHANGELOG.en-US.md [hunk 1]
```diff
@@ -15,6 +15,20 @@ timeline: true
 
 ---
 
+## 3.26.1
+
+`2019-12-09`
+
+- 🐞 Fix List pagniation options text align.
```
Comment (zombieJ) [inline CHANGELOG.en-US.md]: Ref?

File: CHANGELOG.en-US.md [hunk 2]
```diff
@@ -15,6 +15,20 @@ timeline: true
...
+- 🐞 Fix Button text not align well in Chrome. [#20059](https://github.com/ant-design/ant-design/pull/20059)
+- 🐞 Fix DescriptionItem `className` error applied to label. [#20067](https://github.com/ant-design/ant-design/pull/20067) [@Liu-Ya](https://github.com/Liu-Ya)
+- 🐞 Fix Tree duplicated icons when showLine is true. [#20090](https://github.com/ant-design/ant-design/issues/20090)
```
Comment (zombieJ) [inline CHANGELOG.en-US.md]: `showLine`

File: CHANGELOG.en-US.md (hunk 1 above)
Comment (vagusX) [reply CHANGELOG.en-US.md]: pagniation  --> pagination

File: CHANGELOG.zh-CN.md [hunk 3]
```diff
@@ -15,6 +15,20 @@ timeline: true
...
+`2019-12-09`
+
+- 🐞 修复 List 下分页选项文本对齐问题。[#20037](https://github.com/ant-design/ant-design/issues/20037)
+- 🐞 修复 Chrome 下 Button 文字没有垂直居中的问题。[#20059](https://github.com/ant-design/ant-design/pull/20059)
+- 🐞 修复 DescriptionItem 的 `className` 错误应用于 label。[#20067](https://github.com/ant-design/ant-design/pull/20067) [@Liu-Ya](https://github.com/Liu-Ya)
+- 🐞 修复 Tree `showLine` 为 true 时展示多余图标的问题。[#20090](https://github.com/ant-design/ant-design/issues/20090)
+- Table
+  - 🐞 修复小号 Table 表头边框丢失的问题。[#20030](https://github.com/ant-design/ant-design/issues/20030)
+  - 🐞 修复 `filterIcon` 返回字符串或数字时报错的问题。
+  - 🐞 修复 `filterIcon` 返回 Tooltip 时显示了错误的 title。[#20049](https://github.com/ant-design/ant-design/issues/20049)
```
Comment (afc163) [inline CHANGELOG.zh-CN.md]: title 加 `

**Comment by afc163** [comment]
https://github.com/ant-design/ant-design/pull/20118

**Comment by afc163** [comment]
还没发布的话，补一下 https://github.com/ant-design/ant-design/pull/20142

**Pull request merged**